src/main/java/com/epam/prejap/teatrees/TeaTrees.java

```diff
@@ -42,18 +42,23 @@ public Score play() {
     }
 
     public static void main(String[] args) {
+
         int rows = 10;
         int cols = 20;
-        int delay = 500;
-
+        int delay = 5;
+        final String nameJSONFile = "score_list.json";
```

Comment:
```suggestion
        final String scoreListFileName = "score_list.json";
```

JSONFile name is meaningless, doesn't tell what this is really about.